cases/src/org/commcare/cases/util/CasePurgeFilter.java

```diff
@@ -102,29 +105,20 @@ private void setIdsToRemoveNew(IStorageUtilityIndexed<Case> caseStorage, Vector<
                     indexHolder.addElement(index);
                 }
             }
-            String nodeStatus;
-            //Four cases, applied in order. One and two: closed or unowned it starts life dead
-            if (!owned || c.isClosed()) {
-                nodeStatus = STATUS_DEAD;
-            } else {
-                //Otherwise we need to see whether this case maintains any extension indices,
-                //if so, it's abandoned. Otherwise it's Alive
-                boolean abandoned = false;
-                for (CaseIndex index : indexHolder) {
-                    if (index.getRelationship().equals(CaseIndex.RELATIONSHIP_EXTENSION)) {
-                        abandoned = true;
-                        break;
-                    }
-                }
-                if (abandoned) {
-                    nodeStatus = STATUS_ABANDONED;
-                } else {
-                    nodeStatus = STATUS_LIVE;
-                }
+            int nodeStatus = 0;
+            if(owned) {
+                nodeStatus |= STATUS_OWNED;
             }
 
+            if(!c.isClosed()) {
+                nodeStatus |= STATUS_OPEN;
+            }
+
+            if(owned && !c.isClosed()) {
+                nodeStatus |= STATUS_RELEVANT;
+            }
```

Comment:
perfect place for a `private static int calculateCaseStatus(boolean isOwned, boolean isOpen)` helper method